parts/chapters/subsections/4.3/MESSAGES.fodt

```diff
@@ -4506,7 +4506,7 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
       <table:covered-table-cell/>
      </table:table-row>
     </table:table>
-    <text:p text:style-name="P14560">Table <text:sequence text:ref-name="refTable15" text:name="Table" text:formula="ooow:Table+1" style:num-format="1">4.5</text:sequence>: MESSAGES Keyword Description</text:p>
+    <text:p text:style-name="P14560">Table <text:sequence text:ref-name="REF_TABLE_MESSAGES_KEYWORD__CRIPTION_4_3" text:name="Table" text:formula="ooow:Table+1" style:num-format="1">4.5</text:sequence>: MESSAGES Keyword Description</text:p>
```

Comment:
Did you do the renaming of text:ref-name?

Reply:
Yes I tried to create a unique ref-name for the table number. But it didn't work.